Portfolio/index.html

```diff
@@ -18,7 +18,7 @@ <h1>Develop</h1>
             <li>Reviews</li>

```

Comment:
Consider using a more descriptive alt attribute for image tags to improve accessibility. For example, 'Phone icon'